### PR TITLE
ruixi/build-ios-tests

### PR DESCRIPTION
**Pull request by ruixi1994**
This change serves for build MSAL iOS on cpp repo.  Changes includes minor fix and moving of files which complies for mac or ios to corresponding folders

### CODE REVIEW

File: IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m [hunk 1]
```diff
@@ -66,12 +66,6 @@ - (instancetype)initWithURL:(NSURL *)url
             __auto_type config = [SFSafariViewControllerConfiguration new];
             _safariViewController = [[SFSafariViewController alloc] initWithURL:url configuration:config];
         }
-#if !TARGET_OS_MACCATALYST
```
Comment (oldalton) [inline IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m]: what is the reason for this change?

Comment (ruixi1994) [reply IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m]: This method is already deprecated when I build it with CMake. So I removed it and seems not causing any issue for build in common repo there.  Let me know if you have concerns. Thanks!

Comment (oldalton) [reply IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m]: We still support iOS 10 where alternative method is unavailable, so we should keep this.

**Comment by oldalton** [comment]
SSO extension + macOS issues fixed in https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/631

File: IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m (hunk 1 above)
Comment (ruixi1994) [reply IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m]: Reverted this change.
